ftplugin/java.vim

```diff
@@ -0,0 +1,20 @@
+vim9script
+
+def OutlinePreProcessInternal(outline: list<string>): list<string>
+    #javadoc string removal TODO
+    var i = 0
+    var is_javadoc = false
+
+    var tmp_string = "<hshnnTejwqik93la,AMK3N2MNMAKPD+03mn2nhkalpdpk3nsla>"
+    for item in outline
+        i = i + 1
+    endfor
+    return outline
+enddef
+
+import autoload "../lib/ftfunctions/java.vim"
+
+b:OutlinePreProcessInternal = OutlinePreProcessInternal
```

Comment:
The `OutlinePreProcessInternal( ... )` function could be safely removed along with `b:OutlinePreProcessInternal = OutlinePreProcessInternal. `

Reply:
applied this suggestion in latest commit, please review